utils.py

```diff
@@ -10,15 +10,33 @@
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 from webhook import DiscordWebhook, DiscordEmbed
 from chromedriver_py import binary_path as driver_path
-import json, platform, darkdetect, random, settings, threading, hashlib, base64
+import json, platform, darkdetect, random, settings, threading, hashlib, base64, string
 normal_color = Fore.CYAN
-e_key = "YnJ1aG1vbWVudA==".encode() #TODO: this needs to become an app setting the user sets. On app start could just generate a key randomly too. 
+
+def write_data(path,data):
+    with open(path, "w") as file:
+        json.dump(data, file)
+    file.close()
+
+#TODO: Enable this as an app setting for user to choose their own optional key & regenerate button
+try:
+  with open("./data/vault.json","r") as file:
+      keys = json.load(file)
+  file.close()
+except FileNotFoundError:
+  generateKeySecret = "".join(random.choices(string.ascii_letters + string.digits, k=16))
+  write_data("./data/vault.json",[{ "generated_key_secret": generateKeySecret }])
+  with open("./data/vault.json","r") as file:
+      keys = json.load(file)
+  file.close()
```

Comment:
since you are using a context manager for the file open (`with open()`) you do not need to explicitly close the file. This is automatically done when the with exits and thus this line can be deleted